apps/src/blockly/types.ts

```diff
@@ -87,7 +86,7 @@ export interface BlocklyWrapperType extends GoogleBlocklyType {
   levelBlockIds: string[];
   isStartMode: boolean;
   isToolboxMode: boolean;
-  toolboxBlocks: Element | ToolboxDefinition | undefined;
+  toolboxBlocks: string | undefined;
```

Comment:
Does this cause any problems for Music Lab where toolboxes are defined in JSON? Did the imported `ToolboxDefinition` type  cover this?

From the browser console on a music lab page:
![image](https://github.com/code-dot-org/code-dot-org/assets/43474485/52ca766e-3c7f-4a2b-b303-055d147db9b3)

Reply:
Good call out, I will double check this.

Reply:
I don't think it causes issues but it is incorrect so I'll update this back.

Reply:
I changed it to be just `ToolboxDefinition | undefined` because `ToolboxDefinition` includes `Node` as a possible type.

Reply:
I was thinking the same thing!